spaceship/lib/spaceship/test_flight/client.rb

```diff
@@ -112,15 +112,21 @@ def add_group_to_build(app_id: nil, group_id: nil, build_id: nil)
     ##
 
     def post_tester(app_id: nil, tester: nil)
-      assert_required_params(__method__, binding)
+      return create_app_level_tester(app_id: app_id,
```

Comment:
I like the change below where the client does not need to know about testers, and just deals with parameters or data hashes, instead of model objects.

I would also agree that it should be the responsibility of `Spaceship::TestFlight::Tester` to call the method with the correct parameters. The proxy method is be unnecessary. I assume this will require changing some tests 👍